### ISSUE TITLE
Why does epochs stop at 56 times?

### ISSUE THREAD

**Issue by taotaotao3** [raghakot/keras-resnet#63]
I'm building a GPU with win10 and jupyter notebook.
Even though epoch is set to 200, it stops at 56 times. Why. There is no error wording.
=====
Epoch 56/200
1562/1562 [=======================]-46s 29ms / step-loss: 0.3580-acc: 0.9366-val_loss : 0.6319-val_acc: 0.8615
In []:

**Comment by jiaaihhy** [comment]
m
